### PR TITLE
View All Meetings Component 

### PR DESCRIPTION
**Pull request by lgallo95**
## Description

Adding the function to view appointments on the calendar. Dummy data is working and all the appointments are being listed. The next thing we need is to fix the back end or front end so the dates are working correctly.

Co authors:
Nelson (Bryce) Slade
Morgan Jones
Arison Arias
Jordan Chaz Cabanada
Vernon Bengtson

#### Video Link

https://www.loom.com/share/06e2cca610e84b2f920dca9bbf506852

#### Trello Link

<blockquote class="trello-card"><a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;ID37TYFd&#x2F;240-as-an-admin-i-want-to-be-able-to-view-all-scheduled-meetings">As an admin, I want to be able to view all scheduled meetings.</a></blockquote><script src="https://p.trellocdn.com/embed.min.js"></script>

## Type of change

___Please delete options that are not relevant.___

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] This change requires a documentation update

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have removed unnecessary comments/console logs from my code
- [x] My changes generate no new warnings
- [x] I have checked my code and corrected any misspellings
- [x] No duplicate code left within changed files
- [x] Size of pull request kept to a minimum
- [x] Pull request description clearly describes changes made & motivations for said changes

### CODE REVIEW

File: src/components/pages/ViewAllMeetings/MeetingInfo.js [hunk 1]
```diff
@@ -1,10 +1,15 @@
 import React from 'react';
+import './MeetingInfo.css';
 
 const MeetingInfo = props => {
   const { info } = props;
 
+  console.log(props);
```
Comment (phillipgalbraith) [inline src/components/pages/ViewAllMeetings/MeetingInfo.js:7]: We often use these console logs until we are production-ready, so I guess we are just keeping an eye on these at this point.

**Review by phillipgalbraith — APPROVED** [review]
Why is the topic not repeated in the Modal?  It could help to maintain the hierarchy of the info in the event.  However the data displayed is organized well! I am glad to see the intent to keep styles in a separate files and use classes for separation of concerns.  This is well scoped.

**Review by areccus — APPROVED** [review]
This looks really good, I think putting the meetings in the calendar was the best choice.

**Review by ChristianSmith2394 — APPROVED** [review]
Looks great, well written. Keep up the good work!

**Review by OrdainedRat — APPROVED** [review]
Really great addition!

**Review by Astrospence — APPROVED** [review]
Clear description and clear, meaningful commits. Great work.

**Review by arsalan2243 — APPROVED** [review]
wow this is great. being able to check the appointments on calendar is one of the most important functionality features in my opinion.

File: src/components/pages/ViewAllMeetings/VAMC.js [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+
+function VAMC(props) {
+  const { meetingData } = props;
+
+  const [isModalVisible, setIsModalVisible] = useState(false);
+  const [isScheduleModalVisible, setIsScheduleModalVisible] = useState(false);
+  const [event, setEvent] = useState(null);
+  const [eventsArr, setEventsArr] = useState(initialValues);
+
+  // useEffect(() => {
```
Comment (fletchulence) [inline src/components/pages/ViewAllMeetings/VAMC.js:73]: why is this useEffect commented out? Should it just be removed?

File: src/components/pages/ViewAllMeetings/ViewAllMeetings.js [hunk 3]
```diff
@@ -1,23 +1,10 @@
-import React, { useState, useEffect } from 'react';
-import axiosWithAuth from '../../../utils/axiosWithAuth';
```
Comment (fletchulence) [inline src/components/pages/ViewAllMeetings/ViewAllMeetings.js:2]: why is everything here commented out? Is this still a WIP?

File: src/index.js [hunk 4]
```diff
@@ -18,11 +18,8 @@ import { config } from './utils/oktaConfig';
 import Signup from './components/pages/RoleSignup/Signup';
 import Mentee from './components/pages/RoleSignup/Applications/Mentee';
 import Mentor from './components/pages/RoleSignup/Applications/Mentor';
-
-import AppError from './components/pages/RoleSignup/Applications/AppError';
+import AppSuccess from './components/pages/RoleSignup/Applications/AppSuccess';
```
Comment (fletchulence) [inline src/index.js:21]: Thank you for finally changing this!! I dont know why it is changed in here though... Does it have anything to do with viewing all meetings directly? 
If not `git add <file_name>` might help add only desired files to your PR. Or you can use the GUI in VSCode

File: package.json [hunk 5]
```diff
@@ -11,7 +11,7 @@
     "@craco/craco": "^6.0.0",
     "@okta/okta-react": "^3.0.2",
     "@okta/okta-signin-widget": "^4.1.2",
-    "antd": "^4.18.4",
```
Comment (fletchulence) [inline package.json:14]: oof -- hate to see an update happen just as you push new code. Bad luck

**Review by fletchulence — DISMISSED** [review]
I feel like this might still be in WIP phases, which is fine if it is! If not then please make some changes before pulling this into main

File: package.json (hunk 5 above)
Comment (lgallo95) [reply package.json:14]: rip

File: src/components/pages/ViewAllMeetings/VAMC.js (hunk 2 above)
Comment (lgallo95) [reply src/components/pages/ViewAllMeetings/VAMC.js:73]: We will be using it later. Just need to get the data correct.

File: src/index.js (hunk 4 above)
Comment (lgallo95) [reply src/index.js:21]: What are you referring to?

**Review by jcpcab — APPROVED** [review]
Great work before we plug in actual data from backend!

**Review by WaylayOvercast — APPROVED** [review]
looks really good, putting the meetings in the calendar makes sense.

**Review by fletchulence — COMMENTED** [review]
Nicely done here! Very glad we got this in. Vern taking over from here should be a piece of cake!

**Pull request merged**